broker-core/src/test/java/io/zeebe/broker/workflow/WorkflowInstanceTokenTest.java

```diff
@@ -326,8 +332,7 @@ public void shouldCompleteInstanceAfterNonInterruptingBoundaryEventTriggered() {
         testClient.createWorkflowInstance(r -> r.setBpmnProcessId(processId)).getInstanceKey();
 
     // when
-    testClient.receiveElementInState(
-        workflowInstanceKey, "task-1", WorkflowInstanceIntent.ELEMENT_ACTIVATED);
+    testClient.receiveTimerRecords().withIntent(TimerIntent.CREATED).getFirst();
```

Comment:
In a class rule, there could be multiple timer records. 
```suggestion
   testClient
        .receiveJobs()
        .withWorkflowInstanceKey(workflowInstanceKey)
        .withIntent(JobIntent.CREATED)
        .getFirst();
```